float64 distance_x
float64 distance_y
std_msgs/String user
---
bool success
